src/Domain/DonationRepository.php

```diff
@@ -406,7 +406,7 @@ public function findWithExpiredMatching(): array
             ->leftJoin('d.fundingWithdrawals', 'fw')
             ->where('d.donationStatus = :expireWithStatus')
             ->andWhere('d.createdAt < :expireBefore')
-            ->groupBy('d')
+            ->groupBy('d.id')
```

Comment:
This looks fine but is there any chance you can test it works fine locally?

Reply:
I have run the command locally using 
```
# SLACK_API_TOKEN=xoxb-token SLACK_CHANNEL=channel php matchbot-cli.php matchbot:expire-match-funds
```

and it seems to run as normal.